package.json

```diff
@@ -21,14 +21,18 @@
   "dependencies": {
     "react": "16.7.0",
     "react-dom": "16.7.0",
-    "react-router-dom": "4.3.1"
+    "react-redux": "6.0.0",
+    "redux": "4.0.1",
+    "redux-thunk": "2.3.0"
   },
   "devDependencies": {
+    "babel-plugin-transform-object-rest-spread": "6.26.0",
+    "html-webpack-plugin": "3.2.0",
+    "react-router-dom": "4.3.1",
```

Comment:
I think react-router-dom package should be in the dependencies and not the devDependencies.

Reply:
Yes please